### DISCUSSION TITLE
is this a compatibility issue?

### DISCUSSION THREAD

**Discussion by WolfLiSteel** [TownyAdvanced/Towny#6444] [Q&A] [answered]
I get this everything i try to enable towny chat

! [Thu 12:47:32 WARN  Minecraft] Enabled plugin with unregistered PluginClassLoader TownyChat v0.101
- [Thu 12:47:32 ERROR TownyChat] Towny version does not meet required minimum version: 0.98.5.0
- [Thu 12:47:32 ERROR TownyChat] Disabling TownyChat...
  [Thu 12:47:32 INFO  TownyChat] Disabling TownyChat v0.101
  [Thu 12:47:32 INFO  WolfyUtilities] Dependencies Loaded. Calling DependenciesLoadedEvent!
 
  [Thu 12:47:32 INFO  CustomCrafting] ------------------------------------------------------------------------
  [Thu 12:47:32 INFO  CustomCrafting] Loading Recipes & Items
  [Thu 12:47:32 INFO  CustomCrafting] - - - - [Local Storage] - - - -
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL] Looking through data folder...
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL] Loading Items
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL] Loading Recipes
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL] Loaded 0 recipes; Skipped: 0 error/s, 2 already existing
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL_OLD] Loaded 0 recipes; Skipped: 0 error/s, 0 already existing
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL_LEGACY] Loaded 0 recipes; Skipped: 0 error/s, 0 already existing
  [Thu 12:47:32 INFO  CustomCrafting] [LOCAL] Loaded 2 recipes
  [Thu 12:47:32 INFO  CustomCrafting] Indexing Recipe Book...
  [Thu 12:47:32 INFO  CustomCrafting] Indexed Recipe Book!
  [Thu 12:47:32 INFO  CustomCrafting] ------------------------------------------------------------------------

**Comment by WolfLiSteel** [comment]
im using Towny-0.98.6.0.jar btw

**Comment by LlmDl** [comment, marked as answer]
File a ticket and don't skip the form.
